web/src/copied_tooltip.js

```diff
@@ -0,0 +1,27 @@
+import tippy from "tippy.js";
+
+import {$t} from "./i18n";
+
+export function show_copied_confirmation($copy_button, on_hide_callback, timeout_in_ms = 1000) {
```

Comment:
Do not name a variable with a `$` prefix unless it is a jQuery object; an `HTMLElement` is not a jQuery object. This naming convention is not just for readability; it’s also important for the security checks of [eslint-plugin-no-jquery](https://github.com/wikimedia/eslint-plugin-no-jquery).